build-packer-image.sh

```diff
@@ -2,7 +2,7 @@
 
 set -euo pipefail
 
-TAG=v6
+TAG=v8
```

Comment:
Can we use v7 now? As we understand the mechanics below now. The v7 wasn't used yet, right?